proto/src/determined/user/v1/user.proto

```diff
@@ -39,6 +39,10 @@ message PatchUser {
   optional string username = 5;
   // Name to display in the web UI.
   optional string display_name = 6;
+  // Password of user.
+  optional string password = 7;
+  // Indicates if the password is pre-salted or not.
```

Comment:
This comment should be clarified. `is_hashed` indicates if the password field is sent salted and hashed